koji_containerbuild/plugins/builder_containerbuild.py

```diff
@@ -532,7 +536,12 @@ def sigint_handler(*args, **kwargs):
 
         self.logger.info("Response status: %r", response.is_succeeded())
 
-        if response.is_failed():
+        if response.is_cancelled():
+            this_task.cancel()
+            # FIXME - What kind of information do we want in here?
```

Comment:
I think this is as much as you can provide already, since we don't get to supply a reason when cancelling.

The only suggestion I have otherwise is to include a speculation in the message: " (maybe by automated rebuild?)".